### ISSUE TITLE
Recommend to remove Desura store prices / links

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#336]
Desura's parent filed for bankruptcy back in June 2015. (Source: http://www.polygon.com/2015/6/5/8736595/desuras-parent-company-files-for-bankruptcy-ceo-remains-silent )

This has affected Desura, and they are now unable to deliver newly purchased games to their customers.

I've recently tried to purchase a game from Desura, and they have been unable to provide: a confirmation email to complete my registration, a receipt for the purchase / payment of the game, and the email with the steam key / link for the game.

I was lead to the Desura site through ITAD, and I think that Desura price listings should be removed from ITAD before more people get scammed and lose money.

I've also posted this issue to the Enhanced Steam GIT page, since Enhanced Steam appears to use ITAD to get price listings.

[Issue created by circuitbent: 2015-10-22]
[Last updated on bitbucket: 2019-02-02]

[Comment created by Black3ird: 2015-10-29]
I'm sorry you lose money to them. But it was a rush since you didn't even complete and confirm your registration (which new users can't) to Desura before buying the game disregarding research part. And Bad Juju is indeed a Bad company...

[Comment created by circuitbent: 2015-10-28]
I'm not blaming ITAD for myself being scammed out of money. I'm actually trying to help improve ITAD, and also prevent more people from being led to Desura to be scammed. If you already have a Desura account and are familiar with their service, then you probably know their recent history, and know what to look out for. Good for you, but this doesn't help prevent new customers from being scammed. Just because I didn't go on Reddit to "research" Desura, doesn't mean that I rushed and that this is my fault. The blame falls solely on Desura (Bad Juju). Blaming the victims will not solve the problem.

TL;DR Regardless of their reasons or intentions, Desura is currently scamming customers out of money. Don't pull any more pricing data from them until they fix the problem. Reddit is not research. ITAD is cool.

[Comment created by Black3ird: 2015-10-28]
Desura is closed for new customers (search reddit) whereas old customers can enter and use the site. I just bough a dollar game (gearcrack) and they provided both the game and the Steam key. Since you're a new customer, you hit a broken wall but you should have done your searches instead of blaming on ITAD. 

They did not shutdown entirely like some other sites. No one knows for sure the future of Desura since it saw a takeover before. So I suggest first WARNING customers if they do NOT have an account do not go there. If not good enough then HIDING Desura prices instead of "removing" until we all are sure that Desura is closed. Since once a data is gone it is gone and collecting price history data is a tedious job.

TL;DR he's a new customer who rushed a buy before doing research. site works for old customers. any precaution except delete is fine.
